docker/compose-services.yml

```diff
@@ -17,3 +17,20 @@ services:
       - 9000:9000
       - 2500:2500
       - 1100:1100
+
+  minio:
+    image: minio/minio
+    container_name: s3
+    ports:
```

Comment:
Maybe we could have 2 comments such as:

- 9002:9002 # API
- 9001:9001 # Console

Not super important, so you can disregard it. I just thought that it might help people quickly identify which one is what.

Reply:
Going to maintain it like this for now, else we will have to add comments to all the other ports for the mailserver and the database